### DISCUSSION TITLE
How do I set up MindGeekAPI.ini?

### DISCUSSION THREAD

**Discussion by ghost** [stashapp/stash#2854] [Q&A]
I know I'm suppose to enter a website to scrape but I don't know what format to put the site in

**Comment by ghost** [comment]


**Reply by ghost** [reply]
Nope, I didn't figure that one out but I did find and alternative solution. I used StashDB, it's the community metadata database

**Comment by MrX292** [comment]
You need to scrape a valid scene URL for the different sites like Brazzers with the URL and then it saves the token for that site in MindGeekAPI.ini so you can scrape scenes for that site without url
